### PR TITLE
Fix-up standalone binary docs.

### PR DESCRIPTION
**Pull request by isidentical**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -215,14 +215,14 @@ $ pacman -Syu httpie
 $ pacman -Syu
 ```
 
-#### Single Binary Executables
+#### Single binary executables
 
-Have a standalone HTTPie executable, when you don't want to go through the full installatin process.
+Have a standalone HTTPie executable when you don't want to go through the full installation process
```
Comment (janbrasna) [inline docs/README.md:220]: @isidentical I'm still trying to wrap my head around the phrasing here, is there something that escapes me from the structure of the methods above it, or the call-to-action should simply be anything like:
`"Get the standalone HTTPie Linux executables when…"` or
`"Use the standalone HTTPie Linux executables when…"`?

Comment (isidentical) [reply docs/README.md:220]: That sounds much better, would you mind creating a PR for it @janbrasna?

Comment (janbrasna) [reply docs/README.md:220]: @isidentical 👍 On it. Decided in favor of plurals (as there are httpie, http & https "executables" in my perception of things; not sure if there's an internal nomenclature or style guide for that), and for "Get" as a call to action. Arriving shortly.

Comment (janbrasna) [reply docs/README.md:220]: refs #1399

**Pull request merged**